File: VB/WindowsApplication3/ViewModels/CustomerViewModel.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports DevExpress.Mvvm
Imports DXSample.Common.ViewModels

Namespace DXSample.MVVM

    Public Class CustomerViewModel
        Inherits BaseViewModel(Of Customer)
        Protected Overrides Sub LoadEntities()
            MyBase.LoadEntities()
            For i As Integer = 0 To 9
                Entities.Add(New Customer() With {.ID = Entities.Count, .Name = String.Format("Test Name {0}", Entities.Count)})
            Next i
        End Sub
    End Class
End Namespace

File: CS/WindowsApplication3/ViewModels/CustomerViewModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using DevExpress.Mvvm;
using DXSample.Common.ViewModels;

namespace DXSample.MVVM {

    public class CustomerViewModel : BaseViewModel<Customer> {
        protected override void LoadEntities() {
            base.LoadEntities();
            for (int i = 0; i < 10; i++) {
                Entities.Add(new Customer() { ID = Entities.Count, Name = string.Format("Test Name {0}", Entities.Count) });
            } 
        }
    }
}
